app/src/debug/java/timur/gilfanov/messenger/debug/DebugDataRepository.kt

```diff
@@ -0,0 +1,15 @@
+package timur.gilfanov.messenger.debug
+
+import kotlinx.coroutines.flow.Flow
+
+interface DebugDataRepository {
+
+    suspend fun getSavedScenario(): DataScenario?
+    suspend fun toggleAutoActivity(enabled: Boolean)
+    suspend fun toggleNotification(show: Boolean)
+    suspend fun updateSettings(transform: (DebugSettings) -> DebugSettings)
+    suspend fun clearAllData()
+    suspend fun regenerateData()
+    suspend fun initializeWithScenario(scenario: DataScenario)
```

Comment:
"initialization" here means population with the data for given scenario? If so, must clear ambiquity of the word "initialization" with more speciefic:
```suggestion
    suspend fun populateWithData(scenario: DataScenario)
```

Reply:
this method not only regenerate data for given scenario but also set scenario to preferences. I think `initializeWithScenario` is a good name here.